localstack-core/localstack/utils/catalog/loader.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+from pydantic import BaseModel
+
+LICENSE_CATALOG_PATH = ""
+
+
+class AwsRemoteCatalog(BaseModel):
+    schema_version: str
+    localstack: dict[str, str]
+    services: dict[str, dict]
+    cloudformation_resources: dict[str, dict]
```

Comment:
nit: We could make the typing here more explicit, as we assume it to be so in subsequent checks. (e.g., direct references to `"methods"`)